roles/pulp_devel/tasks/main.yml

```
@@ -53,6 +53,12 @@
   become: true
   become_user: "{{ developer_user }}"
 
+- name: Set crypto policy for Fedora (to allow using CDN certificates)
+  become: true
+  command: update-crypto-policies --set LEGACY
+  when:
+    - ansible_facts.distribution == "Fedora"
+
```

To make it pass molecule idempotency test (the part of the CI that is failing):
```suggestion
  tags:
    - molecule-idempotence-notest
```